app/helpers/bids_helper.rb

```diff
@@ -6,4 +6,10 @@ def content_for_row(row_number, bid)
       number_to_currency(bid.amount)
     end
   end
+
+  def user_is_winning_bidder?(user, auction)
+    return false if !auction.current_bid?
+
+    user.id == auction.current_bid.bidder_id
+  end
```

Comment:
This is helpful. So something like:

```ruby
class Presenter::Auction
# ...
  user_is_winning_bidder?(user)
    return false if !current_bid?
    user.id == current_bid.bidder_id
  end
end
```

And just call that directly in the view? (`<%=  auction.user_is_winning_bidder?(current_user) %>`)

(I'm really enjoying the aggressive tech debt prevention approach. It keeps growing on me)